task16/index.js

```diff
@@ -261,18 +83,74 @@ const renderTabContainer = (value) => {
 const createArticleTitle = ({ article }) => {
   const articleTitleFragment = document.createDocumentFragment();
   for (let i = 0; i < article.length; i++) {
-    const articleTitleItem = createElementWithClassName("li", "article-title-item");
-    const articleWrapper = createElementWithClassName("article", "article-wrapper");
+    const articleTitleItem = createElementWithClassName(
+      "li",
+      "article-title-item"
+    );
+    const articleWrapper = createElementWithClassName(
+      "article",
+      "article-wrapper"
+    );
     const articleLink = createElementWithClassName("a", "article-link");
     const articleTitle = createElementWithClassName("h1", "article-title");
     articleTitle.textContent = article[i].title;
     articleLink.href = "#";
 
-    articleTitleFragment.appendChild(articleTitleItem).appendChild(articleWrapper).appendChild(articleLink).appendChild(articleTitle);
+    articleTitleFragment
+      .appendChild(articleTitleItem)
+      .appendChild(articleWrapper)
+      .appendChild(articleLink)
+      .appendChild(articleTitle);
+
+    renderNewArrivals(article[i], articleLink);
+    renderCommentCount(article[i], articleLink);
   }
   return articleTitleFragment;
 };
 
+const renderNewArrivals = (article, articleLink) => {
+  const articleDate = elapsedTime(article.date);
+  const newArrivalsLimiter = 3;
+  if (articleDate <= newArrivalsLimiter) {
+    articleLink.appendChild(createNewArrivals());
+  }
+};
+
+const renderCommentCount = (article, articleLink) => {
```

Comment:
IMO
The use of a generic name will allow us to respond to any changes in the addressee.
```suggestion
const renderCommentCount = (article, parent) => {
```

Reply:
You can write like this! I learned a lot. I will study more because the idea of arguments has not taken hold.Thank you for pointing that out.
16a81fe